maps/sccv_horizon/sccv_horizon-2_deck_2.dmm

```diff
@@ -22257,7 +22257,8 @@
 	},
 /obj/structure/sign/double/barsign{
 	dir = 4;
-	pixel_x = -64
+	pixel_x = -64;
+	req_access = list(25)
```

Comment:
I don't think this should be defined? It's the Horizon's bar, so `access_bar` should be fine here.

Reply:
spacebar.dm assumes at first that access_bar is present, but later on checks req_access when it does (check_access(card)).

If this is not the case I will revert my change to deck 2

Reply:
![image](https://user-images.githubusercontent.com/91219575/163228599-32d43641-5875-481e-a3aa-0d2844e5d14a.png)
Confirmed it, if I leave req_access null anyone can access it, this is me changing the sign as janitor.

Reply:
You need to delete it entirely, not set it to null
Having the `req_access` var mapped in on that sign either way overwrites the `req_access = list(access_bar)` in the object's definition, thus letting anyone access it when you set it to null
Any variables mapped in overwrite those set in code